### ISSUE TITLE
Discrepancy between reported and achieved results

### ISSUE THREAD

**Issue by Amrsaeed** [ZhenboSong/mono_velocity#3]
Hello,
Thank you for your work on this.

When I tried using the test script with the provided checkpoint to generate new json file with the predictions, it resulted in a much different result than the one provided. The evaluate script also fails when trying to run on the new json file as the difference between the ground truth and prediction is too big so it is construed as not predicted. Do you have any idea why this might be the cause?

I have also tried re-training but got the same results. One thing worth mentioning though is when training, the best model is saved very early on (epoch 3) with a validation loss of 0.253.
Only thing I changed was updating the flownet packages to accommodate newer cuda and pytorch versions.

Any help would be appreciated.
